builtin/credential/userpass/path_user_policies.go

```diff
@@ -43,9 +48,28 @@ func (b *backend) pathUserPoliciesUpdate(ctx context.Context, req *logical.Reque
 		return nil, fmt.Errorf("username does not exist")
 	}
 
-	userEntry.Policies = policyutil.ParsePolicies(d.Get("policies"))
+	var resp *logical.Response
+
+	policiesRaw, ok := d.GetOk("token_policies")
+	if !ok {
+		policiesRaw, ok = d.GetOk("policies")
+		if ok {
+			userEntry.Policies = policyutil.ParsePolicies(policiesRaw)
+			userEntry.TokenPolicies = nil
+		}
+	} else {
+		userEntry.TokenPolicies = policyutil.ParsePolicies(policiesRaw)
+		_, ok = d.GetOk("policies")
+		if ok {
+			if resp == nil {
+				resp = &logical.Response{}
+			}
+			resp.AddWarning("Both 'token_policies' and deprecated 'policies' values supplied, ignoring the deprecated value")
```

Comment:
I wonder about the 204 vs 200 responses here.  Should we just punt on the warnings and just leave these in documentation.